components/net_traits/request.rs

```diff
@@ -178,7 +176,7 @@ impl Default for RequestInit {
             use_cors_preflight: false,
             credentials_mode: CredentialsMode::Omit,
             use_url_credentials: false,
-            origin: ServoUrl::parse("about:blank").unwrap(),
+            origin: ServoUrl::parse("about:blank").unwrap().origin(),
```

Comment:
I think it shouldn't, this is a [scheme fetch](https://fetch.spec.whatwg.org/#scheme-fetch) and the spec never mentions opaque origins for scheme fetches.

Whether this should be a scheme fetch or not, I don't know :sad:

It's also been worrying me that there's no case of setting opaque origins in Servo.